src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -2195,6 +2197,125 @@ static inline __m128 atan2_ps( __m128 y, __m128 x )
     return result;
 }
 
+static const __m256 _ps_atanrange_hi_avx = _mm256_set1_ps(2.414213562373095);
+static const __m256 _ps_atanrange_lo_avx = _mm256_set1_ps(0.4142135623730950);
+static const __m256 _ps_cephes_PIF_avx = _mm256_set1_ps(3.141592653589793238);
+static const __m256 _ps_cephes_PIO2F_avx = _mm256_set1_ps(1.5707963267948966192);
+static const __m256 _ps_cephes_PIO4F_avx = _mm256_set1_ps(0.7853981633974483096);
+
+static const __m256 _ps_atancof_p0_avx = _mm256_set1_ps(8.05374449538e-2);
+static const __m256 _ps_atancof_p1_avx = _mm256_set1_ps(1.38776856032E-1);
+static const __m256 _ps_atancof_p2_avx = _mm256_set1_ps(1.99777106478E-1);
+static const __m256 _ps_atancof_p3_avx = _mm256_set1_ps(3.33329491539E-1);
+
+static inline __m256 atan_ps( __m256 x )
```

Comment:
Again, all  these changes  on atan_ps in AVX2 are for phase, and not color temperature